src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -419,6 +416,18 @@ class SignUpModalController {
   }
 
   afterRender (context) {
+    // Handle inactivity error
+    // The Okta widget has an issue where if the page is idle for a period of time,
+    // the Okta interaction session will expire, causing the widget to show an error "You have been logged out due to inactivity..."
+    // The user then has to click "back to sign in," to rerender the Okta widget.
+    // To avoid the error showing, we check if the context formName is 'terminal'. If it is, we know there was an error,
+    // and we re-render the widget to refresh the widget.
+    // We also set a flag to prevent the widget from refreshing multiple times.
+    if (context.formName === 'terminal' && !this.refreshedDueToInactivity) {
+      this.refreshedDueToInactivity = true
+      this.reRenderWidget()
```

Comment:
Do we need to set `this.refreshedDueToInactivity` back to `false` at some point?